packages/services/config.mode-docker.yaml

```diff
@@ -0,0 +1,19 @@
+---
+chains:
+  - name: "localhost"
+    id: "31337"
+    rpc:
+      http: "http://host.docker.internal:8545"
```

Comment:
this was a workaround i found for Mac specifically. not sure if this works on any other platform.

https://github.com/docker/compose/issues/3800#issuecomment-496480571